test/ERC4626.t.sol

```diff
@@ -566,6 +717,37 @@ contract SparkVaultWithdrawSuccessTests is SparkVaultTestBase {
         assertEq(asset.balanceOf(randomUser),     assets);
     }
 
+    function test_withdraw_depositCapExceeded() public {
+        uint256 assets = vault.assetsOf(user1);
+
+        assertEq(assets, 1_000_107.459782e6);
+
+        // Deal value accrued to the vault
+        deal(address(asset), address(vault), 1_000_107.459782e6);
+
+        assertEq(vault.totalSupply(),             1_000_000e6);
+        assertEq(vault.balanceOf(user1),          1_000_000e6);
+        assertEq(vault.assetsOf(user1),           assets);
+        assertEq(vault.totalAssets(),             assets);
+        assertEq(asset.balanceOf(address(vault)), assets);
+        assertEq(asset.balanceOf(user1),          0);
+
+        // Set deposit cap to 0
+        vm.prank(admin);
+        vault.setDepositCap(0);
+
```

Comment:
Lets just add a quick line to show they can't deposit but can withdraw, just try depositing 1 unit here.